db/fixtures/ae_datastore/ManageIQ/System/Event/EmsEvent/RHEVM.class/network_add_vm_interface.yaml

```diff
@@ -9,4 +9,4 @@ object:
     description: 
   fields:
   - rel4:
-      value: "/System/event_handlers/event_action_refresh?target=src_vm"
+      value: "/System/event_handlers/event_action_refresh_new_target?target=src_vm"
```

Comment:
Correct, the issue is when we add new vm with network attached. This would cause 2 events one for vm creation the other for nic being attached to a vm. The second event is processed first which check that vm do not exists and uses ems as target. Mitigation of this situation is to use the same flow as vm creation.